lib/exception/TingClientException.php

```diff
@@ -1,5 +1,21 @@
 <?php
```

Comment:
Missing documentation and who throws the messages. Is it from the data-well it would be nice with some documentation, as these errors is not where to be found in the code base.